admin/tool/policy/lib.php

```diff
@@ -55,7 +55,8 @@ function tool_policy_extend_navigation_user_settings(navigation_node $usersettin
  *
  * @return string The HTML code to insert before the head.
  */
-function tool_policy_before_standard_html_head() {
+function tool_policy_before_standard_html_head()
+{
```

Comment:
This is not how we format function signatures in Moodle. We leave curly brackets as the last character on the signature line.